src/plugins/data/public/ui/query_editor/editors/default_editor/index.tsx

```diff
@@ -27,7 +27,7 @@ interface DefaultInputProps extends React.JSX.IntrinsicAttributes {
   provideCompletionItems: monaco.languages.CompletionItemProvider['provideCompletionItems'];
 }
 
-const DefaultInput: React.FC<DefaultInputProps> = ({
+export const DefaultInput: React.FC<DefaultInputProps> = ({
```

Comment:
like above don't need to export completing just make it possible to access the instantied component